pwndbg/aglib/heap/structs.py

```diff
@@ -565,13 +566,33 @@ class c_tcache_perthread_struct_2_30(Structure):
         ("entries", c_pvoid * TCACHE_MAX_BINS),
     ]
 
+class c_tcache_perthread_struct_2_42(Structure):
+    """
+    This class represents the tcache_perthread_struct for 2.42 <= GLIBC as a ctypes struct.
```

Comment:
```suggestion
    This class represents the tcache_perthread_struct for 2.42 >= GLIBC as a ctypes struct.
```
Shall be >= 2.42 right?

Reply:
if you read it left to right it works e.g. 2.42 is less or equal to current GLIBC.
Just wanted one consistent schema e.g. lower limit left and upper limit right

Reply:
oh lol u are right